build/vega-lite-schema.json

```diff
@@ -5146,6 +5146,13 @@
           },
           "type": "array"
         },
+        "init": {
+          "additionalProperties": {
+            "$ref": "#/definitions/SelectionInitArray"
+          },
+          "description": "Initialize the selection with a mapping between field names and arrays of\ninitial values.",
```

Comment:
From your examples in the main conversation page, it seems like `init` supports both (a) a mapping between field names and arrays of initial values and (b) a mapping between channels and arrays of initial values?

If so, how does `init` know if a key is a channel or field name?
(For example, what if my `data` is `[{x:1, y:2}, ...]` -- how do I know if `"x"` is a channel or a field?